src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -118,8 +125,22 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
     }, {
         name: ColumnKey.STATUS,
         render: (trial: IDetailedTrialMatch) => (
-            <div className={styles.statusButton}>
-                <a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}><span className={styles.statusBackground}>{trial.status}</span></a>
+            <div className={styles.statusContainer}>
+                <a target="_blank" href={"https://www.mskcc.org/cancer-care/clinical-trials/" + trial.protocolNo}>
```

Comment:
Why not just use Button from react-bootstrap, and `variant="link"`

Reply:
Some css styles cannot apply for the Button. Besides, this is a link not a button. I don't want to mix them together.